src/actions/PermissionsActions.js

```diff
@@ -23,6 +23,7 @@ export const getCurrentUserLocation = () => {
             })
 
         } catch (error) {
+            console.log(error);
```

Comment:
The permissions errors have been hard to catch. Leaving console.log in here for now to make debugging easier moving forward. We can remove later if desired.